chapter05/drjt/ex2.pl

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+
+# copy-pasted from the chapter (mostly)
+my %total_bytes;
+while ( <> ) {
+  my ( $source, $dest, $bytes ) = split;
+  $total_bytes{$source}{$dest} += $bytes;
```

Comment:
You appear to have overlooked the requirement to ignore any comment lines in the input data.